e2e/tools/validator/validator.yaml

```diff
@@ -1,16 +1,10 @@
 remote:
   host: 192.168.124.28
-  # ### defaults ###
-  # port: 22
-  # username: fedora
-  # password: supersecret
-  # pkey: ~/.ssh/id_rsa
+  pkey: ~/.ssh/kvm
 
 metal:
-  metal_job_name: metal
-  vm_job_name: vm
   vm:
-    pid: 2093543
+    pid: 1819485
```

Comment:
shall this be some fix number such as 0? In my understanding, this will be replaced automatically by the VM process id.

Reply:
I feel we should delete `validator.yaml` and `.gitignore` it so that everyone would make a copy of `validator.yaml.sample`